lib/improver/tests/nowcasting/nowcasting/test_ApplyOrographicEnhancement.py

```diff
@@ -0,0 +1,457 @@
+        self.assertIsInstance(result, iris.cube.Cube)
+        self.assertEqual(result.units, Unit("mm/hr"))
+        self.assertEqual(result.metadata, precip_cube.metadata)
+        self.assertArrayAlmostEqual(result.data, expected0)
+
+    def test_tolerance(self):
+        """Test that the minimum precipitation rate is applied correctly,
+        when the allowed tolerance is altered."""
+        expected0 = np.array([[[[1/32., 1/32., 1/32.],
+                                [1/32., 1/32., 1/32.],
+                                [1/32., 1/32., 1/32.]]]])
+        precip_cube = self.precip_cube.copy()
+        precip_cube.convert_units("mm/hr")
+        precip_cube.data = np.array([[[[-1., -1., 0.],
+                                       [1., 2., 3.],
+                                       [3., 3., 4.]]]])
+        self.precip_cube.convert_units("mm/hr")
+        plugin = ApplyOrographicEnhancement("add")
+        result = plugin.apply_minimum_precip_rate(precip_cube, tolerance=10)
+        self.assertIsInstance(result, iris.cube.Cube)
+        self.assertEqual(result.units, Unit("mm/hr"))
+        self.assertEqual(result.metadata, precip_cube.metadata)
+        self.assertArrayAlmostEqual(result.data, expected0)
+
+    def test_NaN_values(self):
+        """Test a minimum precipitation rate is applied, when NaN values are
+        present within the input."""
+        expected0 = np.array([[[[1/32., 1/32., np.NaN],
+                                [1., 2., np.NaN],
+                                [3., 3., 4.]]]])
+        precip_cube = self.precip_cube.copy()
+        precip_cube.convert_units("mm/hr")
+        precip_cube.data = np.array([[[[-1., -1., np.NaN],
+                                       [1., 2., np.NaN],
+                                       [3., 3., 4.]]]])
+        precip_cube.convert_units("m/s")
+        plugin = ApplyOrographicEnhancement("add")
+        result = plugin.apply_minimum_precip_rate(precip_cube)
+        self.assertIsInstance(result, iris.cube.Cube)
+        self.assertEqual(result.units, Unit("m/s"))
+        self.assertEqual(result.metadata, precip_cube.metadata)
+        result.convert_units("mm/hr")
+        self.assertArrayAlmostEqual(result.data, expected0)
+
+
+class Test_process(IrisTest):
+
+    """Test the apply_orographic_enhancement method."""
+
+    def setUp(self):
+        """Set up cubes for testing."""
+        self.precip_cubes = set_up_precipitation_rate_cube()
+        self.oe_cubes = set_up_orographic_enhancement_cube()
+
+    def test_basic_add(self):
+        """Test the addition of cubelists containing cubes of
+        precipitation rate and orographic enhancement."""
+        expected0 = np.array([[[[0., 1., 2.],
+                                [1., 2., 3.],
+                                [0., 3., 4.]]]])
+        expected1 = np.array([[[[6., 5., 1.],
+                                [6., 5., 1.],
+                                [6., 5., 1.]]]])
```

Comment:
`expected0` and `expected1` are OK here.  Please could you remove the numbers at the end for all other tests.

Reply:
Removed numbers from variable names.